src/test/resources/json/consignments_query_somedata.json

```diff
@@ -0,0 +1,3 @@
+{
+  "query": "{ consignments(limit: 2, currentCursor: \"consignment-ref1\") { pageInfo { startCursor, endCursor, hasNextPage, hasPreviousPage }, edges { node { consignmentid, consignmentReference }}}}"
```

Comment:
Out of curiosity, if you don't include the `pageInfo` in the query, does the response just skip it like regular GraphQL response properties?

Reply:
Yes